cli/lib/kontena/cli/certificate/show_command.rb

```diff
@@ -0,0 +1,23 @@
+require_relative '../services/services_helper'
+
+module Kontena::Cli::Certificate
+  class ShowCommand < Kontena::Command
+    include Kontena::Cli::Common
+    include Kontena::Cli::GridOptions
+
+    parameter "SUBJECT", "Certificate subject"
+
+    requires_current_master
+    requires_current_master_token
+    requires_current_grid
+
+    def show_certificate(certificate)
+      YAML.dump(certificate)
+    end
+
+    def execute
+      certificate = client.get("certificates/#{current_grid}/#{self.subject}")
+      puts YAML.dump(certificate)
```

Comment:
Not sure if we want to dump raw yaml here, do we?

Reply:
most of our `show` commands are outputting pretty yamlish output. hence I thought it might make sense to do it with `Yaml.dump` instead of multiple `puts`. 

In this case there's no need to hide or modify anything from what the API gives.

Reply:
Agreed. I think the formatting of most show commands is mostly identical, and the bits where it differs from the `YAML.dump` are a bit of a pain (like `initial node` vs `initial_member`)

Reply:
> Agreed.

Not sure with what. :) To use `YAML.dump` or NOT to use it?